src/kecleon_bros.c

```diff
@@ -0,0 +1,1777 @@
+extern struct unkStruct_203B224 *gUnknown_203B224;
+extern u16 gUnknown_203B228;
+extern u16 gUnknown_203B22A;
+extern u32 gUnknown_202DE30;
+extern u8 gUnknown_202DE58[];
+extern u8 gUnknown_202E5D8[];
+extern u8 gUnknown_202E1C8[];
+
+extern u8 *gUnknown_80D4920[];
+extern u8 *gUnknown_80D4928[];
+extern u8 *gUnknown_80D4934[];
+extern u8 *gUnknown_80D4970[];
+extern u8 * gUnknown_80D4978[];
+extern u8 *gUnknown_80D4984[];
+extern struct UnkTextStruct2 gUnknown_80DB840;
+extern struct UnkTextStruct2 gUnknown_80DB858;
+extern struct UnkTextStruct2 gUnknown_80DB870;
+extern struct UnkTextStruct2 gUnknown_80DB888;
+extern u8 gUnknown_80DB8A0[];
+extern struct UnkTextStruct2 gUnknown_80DB8B4;
+extern struct UnkTextStruct2 gUnknown_80DB8CC;
+extern u8 gUnknown_80DB8E4[];
+extern u8 gUnknown_80DB8EC[];
+extern struct UnkTextStruct2 gUnknown_80DB8FC;
+extern const struct UnkTextStruct2 gUnknown_80DB914;
+extern u8 gUnknown_80DB92C[];
+extern u8 gUnknown_80DB934[];
+extern struct UnkTextStruct2 gUnknown_80DB944;
+extern struct UnkTextStruct2 gUnknown_80DB95C;
+extern u8 gUnknown_80DB994[];
+extern u8 gUnknown_80DB9A0[];
+extern u8 gUnknown_80DB9A8[];
+extern u8 gUnknown_80DB9B0[];
+
+extern const u8 *gKecleonShopDialogue[KECLEON_BROS_MODE_COUNT][23];
+extern const u8 gTeamToolboxA[];
+extern const u8 gTeamToolboxB[];
+
+// text.s
+extern void sub_80073B8(u32);
+extern void sub_80073E0(u32);
+extern void sub_8007B7C(s32, s32, s32, s32, s32);
+extern void sub_8008C54(u32);
+// code_80130A8.s
+extern void sub_80141B4(const u8 *, u32, struct OpenedFile **, u32);
+extern void sub_8014248(const u8 *, u32, u32, const struct MenuItem *, void *, u32, u32, struct OpenedFile **, u32);
+extern s32 sub_80144A4(s32 *);
+// code_801B3C0.c
+extern u32 sub_801B3C0(struct Item *);
+extern u32 sub_801B410(void);
+extern void sub_801B450(void);
+// code_8098BDC.s
+extern void sub_8099690(u32);
+
+// code_80118A4.c
+extern void PlayMenuSoundEffect(u32);
+extern void PlaySound(u16 songIndex);
+// code_80130A8.s
+extern void xxx_format_and_draw(u32, u32, u8 *, u32, u32);
+
+void sub_8018D30(void);
+void sub_80194F8(void);
+void sub_80195C0(void);
+void sub_8019660(void);
+void sub_8019700(void);
+void sub_8019730(void);
+void sub_8019850(void);
+void sub_80198E8(void);
+void sub_8019944(void);
+void sub_80199CC(void);
+void sub_8019B08(void);
+void sub_8019BBC(void);
+void sub_8019C78(void);
+void sub_8019D30(void);
+void sub_8019D4C(void);
+void sub_8019D68(void);
+void sub_8019DAC(void);
+void sub_8019E04(bool32);
+u32 sub_8019E40(u32);
+u32 sub_8019EDC(u8);
+u8 sub_8019FB0(void);
+void sub_8019FCC(u8);
+void sub_801A010(void);
+void sub_801A064(void);
+void sub_801A0D8(void);
+u32 sub_801A20C(u32);
+u32 sub_801A2A8(u8);
+u8 sub_801A37C(void);
+void sub_801A398(u8);
+void sub_801A3DC(void);
+void sub_801A430(void);
+void sub_801A4A4(void);
+void sub_801A998(void);
+s32 sub_801AE24(u32);
+
+u32 CountKecleonItems(void);
+void Kecleon_SortItems(void);
```

Comment:
Let's swap these two functions to PascalCase too.

Reply:
Addressed